requirements.txt

```diff
@@ -0,0 +1,4 @@
+
+python==3.9.13
```

Comment:
Why using python 3.9.13 instead of the latest version?

Reply:
Because Azure Functions only works with Python 3.9

Reply:
confirmed [https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-python?tabs=asgi%2Capplication-level](url)

Reply:
[teste]([https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-python?tabs=asgi%2Capplication-level](https://github.com/tiagokretzerwalter/wedding-planner/pull/url))

Reply:
[Correct](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-python?tabs=asgi%2Capplication-level)